bin/users

```diff
@@ -10,32 +10,56 @@ const User = require('../app/models/user');
 mongoose.Promise = Bluebird;
 
 function done(err, user) {
-    if (err) throw err;
-    console.log(chalk.green(`Created user ${user.username}`));
-    process.exit(0);
+  if (err) throw err;
+  console.log(chalk.green(`Created user ${user.username}`));
+  process.exit(0);
 }
 
 program
-    .version('1.0.0')
-    .command('create-admin <username> <cardKey>')
-    .description('create an admin user')
-    .action((username, cardKey) => {
-        const mongoURL = process.env.MONGO_URL || 'mongodb://localhost:27017/vote';
-
-        mongoose.connect(mongoURL, connectErr => {
-            if (connectErr) return done(connectErr);
-
-            promptly.password('Enter a password: ', (pwErr, password) => {
-                if (pwErr) return done(pwErr);
-                const user = new User({
-                    username: username,
-                    admin: true,
-                    cardKey: cardKey
-                });
-
-                User.register(user, password).nodeify(done);
-            });
+  .version('1.0.0')
+  .command('create-admin <username> <cardKey>')
+  .description('create an admin user')
+  .action((username, cardKey) => {
+    const mongoURL = process.env.MONGO_URL || 'mongodb://localhost:27017/vote';
+
+    mongoose.connect(mongoURL, connectErr => {
+      if (connectErr) return done(connectErr);
+
+      promptly.password('Enter a password: ', (pwErr, password) => {
+        if (pwErr) return done(pwErr);
+        const user = new User({
+          username: username,
+          admin: true,
+          moderator: true,
+          cardKey: cardKey
+        });
+
+        User.register(user, password).nodeify(done);
+      });
+    });
+  });
+
+program
```

Comment:
This should be generic, more or less 100% duplication

Reply:
Yehh, i had that as one of them `TODO`s :) I have one more just like it for creating testusers